pam/integration-tests/vhs-helpers_test.go

```diff
@@ -322,3 +330,23 @@ func evaluateTapeVariables(t *testing.T, tapeString string, td tapeData) string
 
 	return tapeString
 }
+
+func requireRunnerResultForUser(t *testing.T, sessionMode authd.SessionMode, user, goldenContent string) {
+	t.Helper()
+
+	goldenLines := strings.Split(goldenContent, "\n")
+	goldenContent = strings.Join(goldenLines[max(0, len(goldenLines)-50):], "\n")
```

Comment:
Does that comment correctly describe the intent?

```suggestion
	// Only check the last 50 lines of the golden file, because that's where
	// the result is printed, and printing the full output on failure is too much.
	goldenLines := strings.Split(goldenContent, "\n")
	goldenContent = strings.Join(goldenLines[max(0, len(goldenLines)-50):], "\n")
```

Reply:
What do you mean? :thinking:

Reply:
I mean does the comment I'm suggesting to add correctly explain the following lines? If so, lets add it please.

Reply:
And if not, maybe you can add a comment which correctly explains why we only check the last 50 lines.